CMakeLists.txt

```diff
@@ -53,6 +53,9 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib) # Put libraries into
 set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -march=native")
 set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -march=native")
 
+# auto_ptr was removed in clang 5 so we need to tell boost to not include it
+add_definitions(-DBOOST_NO_AUTO_PTR)
```

Comment:
Any specific reason why this is necessary, like, does the hyrise build break without it? I'd rather keep the CMake code as reduced as possible.

Reply:
we added this line because the build broke on mac for us. can you check if this is still necessary @apopiak ?